resources/views/components/base/button/submit.blade.php

```diff
@@ -4,7 +4,6 @@
     ]) }}
     :disabled="!isValid || isSending"
 >
-    {{ svg('fas-' . ($icon ?? 'paper-plane'), 'icon') }}
```

Comment:
Removed this icon for the time being. Maybe it makes sense to re-add an icon slot or something later. The icon was artificially increasing the button height.

Reply:
Is it about the icon or the layout? I preferred the button with the icon.

Reply:
It's about the layout. I like the icon.

Moving the spinner to the button is a good idea, as the spinner itself also messes with the layout.

The problem is if you have a button with an icon and a button without an icon sitting next to each other, the button with the icon is taller because the icon is increasing its height. Here's a contrived example:

![Screenshot 2024-03-16 at 8 45 51 AM](https://github.com/RetroAchievements/RAWeb/assets/3984985/25952659-8bf1-4ee8-8444-e8c87845394a)

I've tried wrapping the icon in a span and using all sorts of classes like h-*, max-h-*, text-* to change its height, but the icon component appears to be resistant to these efforts.

Reply:
got it. so if i undrrstood correctly with a slot in a separate PR we could make it so all buttons have the same height regardless of whether an icon is displayed or not?

Reply:
I think so. At a minimum, we'll need to track down what specifically is causing the icon to artificially increase the button height. For some reason, all the classnames I threw at it in an attempt to resolve the issue had no effect.

The nuclear option would be to explicitly tweak the button height itself, but that would definitely have a negative impact on the flexibility of the reusable component.

Reply:
yeah, makes sense. thanks for looking into it! let's address this in a separate step